requirements.txt

```diff
@@ -0,0 +1,162 @@
+altair==4.2.0
```

Comment:
This requirements file has packages we do not need that is why we use the virtual environemnt